src/js/service/ImportService.js

```diff
@@ -0,0 +1,121 @@
+/* Image and Animation import service supporting the import dialogs,
+ * file dropper service, and headless import API. */
+(function () {
+  /**
+   * @param {!PiskelController} piskelController
```

Comment:
We are very bad at JSdoc for Piskel, and I'm all for improving this! 
I haven't seen the exclamation mark (!) pattern before. Where is this coming from, what does it mean?